dbms/src/Interpreters/Context.h

```diff
@@ -190,6 +190,9 @@ class Context
     }};
     using DatabasePtr = std::shared_ptr<IDatabase>;
     using Databases = std::map<String, std::shared_ptr<IDatabase>>;
+
+    std::unordered_set<uint64_t> store_id_blocklist;
```

Comment:
It should be a member of the `ContextShared` rather than `Context`.
`ContextShared` is set in the `global_context` and shared by all connections to the tiflash instance. While `Context` is a session level instance that these variable will be copied from the global_context and can be overwritten

Reply:
Fixed